app/controllers/jobseekers/job_applications_controller.rb

```diff
@@ -1,11 +1,15 @@
 class Jobseekers::JobApplicationsController < Jobseekers::BaseController
   include QualificationFormConcerns
+  include Jobseekers::Wizardable
 
   before_action :raise_unless_vacancy_enable_job_applications,
                 :redirect_if_job_application_exists, only: %i[new create new_quick_apply quick_apply]
   before_action :redirect_unless_draft_job_application, only: %i[review]
 
   helper_method :job_application, :qualification_form_param_key, :review_form, :vacancy, :withdraw_form
+  helper_method :personal_details_fields, :professional_status_fields, :qualifications_fields, :employment_history_fields,
```

Comment:
Evidently it's too late in the day for me to make sense of the world, but why do these need to be made helpers?

Reply:
They are `send`ed in app/helpers/job_application_helper.rb#L86

Reply:
Or maybe not... I'll try removing them and see if tests fail.

Reply:
Reader, they did not.